src/main/java/org/ieeervce/gatekeeper/Entity/ReviewLog.java

```diff
@@ -0,0 +1,24 @@
+package org.ieeervce.gatekeeper.Entity;
+import jakarta.persistence.*;
+
+@Entity
+
+public class ReviewLog {
+
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Id
+    private Long id;
+    @Column(nullable = false)
```

Comment:
I believe `@ManyToOne` would be the appropriate one

Similar to one to one functions, `@OneToOne` implies a single user can have a unique review log, *and* vice versa.

Since we're going with a user can post many review logs, `@ManyToOne` would be correct here.

Reply:
oh yes, you are right, anyway need to link it up with User table

Reply:
Yes makes sense